builtin/providers/aws/resource_aws_vpc_peering_connection.go

```diff
@@ -67,7 +67,7 @@ func resourceAwsVPCPeeringCreate(d *schema.ResourceData, meta interface{}) error
 
 	resp, err := conn.CreateVpcPeeringConnection(createOpts)
 	if err != nil {
-		return fmt.Errorf("Error creating vpc peering connection: %s", err)
+		return fmt.Errorf("Error creating VPC Peering Connection: %s", err)
```

Comment:
We probably want to tend towards `errwrap.Wrapf("Error creating VPC Peering Connection: {{err}}", err)` here